src/relay/ir/base.cc

```diff
@@ -66,3 +66,5 @@ TVM_STATIC_IR_FUNCTOR_REGISTER(IRPrinter, vtable)
 
 }  // namespace relay
 }  // namespace tvm
+
```

Comment:
remove

Reply:
here must be a newline at the end of the file as per formatting.